settings.gradle.kts

```diff
@@ -1,19 +1,48 @@
+import com.dropbox.focus.FocusExtension
+
+rootProject.name = "voyager"
+
 enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
 
+pluginManagement {
+    repositories {
+        mavenCentral()
+        google()
+        gradlePluginPortal()
+    }
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.version == null) return@eachPlugin
+            when (requested.id.id) {
+                "com.android.tools.build" -> {
+                    useModule("${requested.id.id}:gradle:${requested.version}")
+                }
+            }
+        }
+    }
```

Comment:
Any reason why this is needed?

Reply:
By default Gradle to resolve plugin artifact by plugin id uses plugin id as group id and add `.gradle.plugin` suffix to plugin id to construct artifact id. So plugin `"pluginId" version "1.0.0"` becomes `"pluginId:pluginId.gradle.plugin:1.0.0"` artifact. 
But AGP build plugin has custom artifact id which we need to override.